examples/client/getNodeInfo.ts

```diff
@@ -0,0 +1,68 @@
+import { NodeInfo } from '@concordium/common-sdk';
+import { createConcordiumClient } from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn ts-node <path-to-this-file> [options]
+
+  Options
+    --help,     -h  Displays this message
+    --block,    -b  A block to query from, defaults to last final block
+    --endpoint, -e  Specify endpoint of the form "address:port", defaults to localhost:20000
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            account: {
+                type: 'string',
+                alias: 'a',
+                isRequired: true,
+            },
+            endpoint: {
+                type: 'string',
+                alias: 'e',
+                default: 'localhost:20000',
+            },
+            block: {
+                type: 'string',
+                alias: 'b',
+                default: '', // This defaults to LastFinal
+            },
+        },
+    }
+);
+
+const [address, port] = cli.flags.endpoint.split(':');
+const client = createConcordiumClient(
+    address,
+    Number(port),
+    credentials.createInsecure(),
+    { timeout: 15000 }
+);
+
+if (cli.flags.h) {
+    cli.showHelp();
+}
+
+/// Get information about the node.
+
+/// The `NodeInfo` includes information of:
+///  - Meta information, such as the, version of the node, type of the node, uptime
+///    and the local time of the node.
+///  - NetworkInfo, which yields data such as the node id, packets sent/received,
+///    average bytes per second sent/received.
+///  - ConsensusInfo. The `ConsensusInfo` returned depends on if the node supports
```

Comment:
```suggestion
///  - ConsensusStatus. The `ConsensusStatus` returned depends on if the node supports
```
I can only find a status, when I try. :thinking:

Reply:
They are the same right? It's just the V2 name that we decided not to use. Still better to keep it consistent with `ConsensusStatus` in the docs as well